batch/create/create_batch_labels_allocation.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2021 Google LLC
```

Comment:
This looks like a new sample but copying an older header. please make sure samples that are new have 2024